scripts/ci/lib.sh

```diff
@@ -1015,19 +1015,21 @@ gate_pr_job() {
 openshift_ci_mods() {
     info "BEGIN OpenShift CI mods"
 
-    info "Env A-Z dump:"
-    env | sort | grep -E '^[A-Z]' || true
+    local debug="${ARTIFACT_DIR:-/tmp}/debug.txt"
+
+    echo "Env A-Z dump:" > "${debug}"
```

Comment:
How about keeping the `info` (which adds a timestamp, right?) and adding 2>&1 here and below?